internal/cli/cmd/login/login.go

```diff
@@ -137,6 +137,11 @@ func (o *loginOptions) run(ctx context.Context) error {
 	var bearerToken, refreshToken string
 	var err error
 
+	currentConfig, err := libConfig.LoadCLIConfig()
+	if err != nil {
+		return fmt.Errorf("error loading current configuration: %w", err)
+	}
+
```

Comment:
This actually shouldn't be necessary. `o.Config` _already_ has the current configuration.

```suggestion
```